bpfman-operator/test/integration/uprobe_test.go

```diff
@@ -0,0 +1,85 @@
+//go:build integration_tests
+// +build integration_tests
+
+package integration
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"regexp"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	uprobeGoCounterKustomize       = "../../../examples/config/default/go-uprobe-counter"
+	uprobeGoCounterUserspaceNs     = "go-uprobe-counter"
+	uprobeGoCounterUserspaceDsName = "go-uprobe-counter-ds"
+	targetKustomize                = "../../../examples/config/default/go-target"
+	targetUserspaceNs              = "go-target"
+	targetUserspaceDsName          = "go-target-ds"
+)
+
+func TestUprobeGoCounter(t *testing.T) {
+	t.Log("deploying target for uprobe counter program")
+	require.NoError(t, clusters.KustomizeDeployForCluster(ctx, env.Cluster(), targetKustomize))
+	addCleanup(func(context.Context) error {
+		cleanupLog("cleaning up target program")
+		return clusters.KustomizeDeleteForCluster(ctx, env.Cluster(), targetKustomize)
+	})
+
+	t.Log("waiting for go target userspace daemon to be available")
+	require.Eventually(t, func() bool {
+		daemon, err := env.Cluster().Client().AppsV1().DaemonSets(targetUserspaceNs).Get(ctx, targetUserspaceDsName, metav1.GetOptions{})
+		require.NoError(t, err)
+		return daemon.Status.DesiredNumberScheduled == daemon.Status.NumberAvailable
+	}, time.Minute, time.Second)
```

Comment:
```suggestion
	},
	// Wait 5 minutes since cosign is slow, https://github.com/bpfman/bpfman/issues/1043
	5 * time.Minute, 10 * time.Second)
```
Ditto